fspiop-api/documents/API-Definition_v1.1.1.md

```diff
@@ -24,6 +24,7 @@ specified types of information.
 |**1.1.1**|2021-09-22|This document version only adds information about optional HTTP headers regarding tracing support in [Table 2](#table-2), see _Distributed Tracing Support for OpenAPI Interoperability_ for more information. There are no changes in any resources as part of this version.|
+|**1.2**|2022-01-31|This version introduces a new HTTP call using the method **PATCH** on **/transfers/**_{ID}_**/error** to be used when there is an internal validation or processing error (not schema validation) during handling of the **PUT /transfers/**_{ID}_ sent from the Payee FSP, both when RESERVED and COMMITTED is used as transferState. This is based on the acceptance of the change request: https://github.com/mojaloop/mojaloop-specification/issues/103|
```

Comment:
Based on this text, I would assume that the earlier **PATCH /transfers/**_{ID}_ notification was somehow changed, as using `transferState` `RESERVED` is mentioned in the same context as `COMMITTED`. The commit notification from version 1.1 should not change in version 1.2, see https://github.com/mojaloop/mojaloop-specification/issues/103#issuecomment-970407712.

Version 1.2 should only add the _possibility_ for a Switch to send the **PATCH /transfers/**_{ID}_/error notification when `COMMITTED` is used as `transferState`.

Reply:
Fixed - this applies only for the transferState COMMITTED (for 1.2)